backend/NycBasics/models.py

```diff
@@ -4,7 +4,7 @@
 class User(models.Model):
     username = models.CharField(max_length=255, null=False, unique=True)
     email = models.EmailField(max_length=255, null=False)
-    password = models.CharField(max_length=50)
+    password = models.CharField(max_length=500)
```

Comment:
why do we have a 500 character limit? is it necessary?

Reply:
is it because the hash has a bigger string length?

Reply:
yes

Reply:
bigger length for hash